ml/tests/test_determinism.py

```
@@ -0,0 +1,221 @@
+            "image_size": 224,
+            "val_split": 0.15,
+            "test_split": 0.15,
+            "seed": SEED,
+        },
+        # `load_config` guarantees this key, defaulting it to True, so a stub
+        # standing in for it has to supply it too.
+        "training": {"deterministic_ops": True},
+        "export": {"output_dir": str(tmp_path / "models")},
+    }
+
+    monkeypatch.setattr(train_module, "load_config", lambda path=None: cfg)
```

Ruff (0.16.1)

[warning] Unused lambda argument: `path`

(ARG005)